RelationalAI.Test/DatabaseTest.cs

```diff
@@ -5,25 +5,31 @@
 
 namespace RelationalAI.Test
 {
+    [Collection("RelationalAI.Test")]
     public class DatabaseTests : UnitTest
     {
+        private readonly EngineFixture engineFixture;
         public static string Uuid = Guid.NewGuid().ToString();
         public static string Dbname = $"csharp-sdk-{Uuid}";
-        public static string EngineName = $"csharp-sdk-{Uuid}";
+
+        public DatabaseTests(EngineFixture fixture)
+        {
+            engineFixture = fixture;
```

Comment:
@miazamrai is this instantiated by xunit ?

Reply:
Yes, the following CollectionDefinition and the Collection annotation on the test classes make it happen
[CollectionDefinition("RelationalAI.Test")]
    public class RelationalAITestCollection : ICollectionFixture<EngineFixture>
    {
        // This class has no code, and is never created. Its purpose is simply
        // to be the place to apply [CollectionDefinition] and all the
        // ICollectionFixture<> interfaces.
    }